hybridse/src/udf/udf.cc

```diff
@@ -157,6 +157,17 @@ int32_t weekofyear(Date *date) {
     }
 }
 
+void int_to_char(int32_t val, StringRef* output){
+    if (val > 256) 
```

Comment:
what if val < 0 ?

Reply:
The api just doesn’t mention about Val<0. If you define the operation it should be, then I’ll glad to have more tests.

Reply:
ok. I will leave it as the same as the number which is bigger than 256. Hope you can satisfy.